src/actions/related-views.ts

```diff
@@ -0,0 +1,6 @@
+import {PlainReduxAction} from './redux-action';
+
+export type RelatedViewAction = HideRelatedView;
+
+export const RELATED_VIEW_HIDE_TOGGLE = 'RELATED_VIEW_HIDE_TOGGLE';
```

Comment:
This is hiding _all_ related view**s** so I wonder if the action should be `RELATED_VIEWS_HIDE_TOGGLE` instead. 

(Similar for `RelatedViewAction` -- however, we may have action that is specific to one kind of related views in the future so I have less strong opinion about the `RelatedViewAction`.)